src/layout/root/RootLayout.jsx

```diff
@@ -6,6 +8,12 @@ export default function RootLayout({ children }) {
   // Example
   return (
     <>
+      <Head>
```

Comment:
Shouldn't it be in the index page, as it is going to be in the entire project?

Reply:
Yes but the index we just put the layout component which will desplay the rest of the pages and components so we don't need to put it in the ulindex just make it in the layout it is a hard-coding way too